src/js/components/covid19/PublicLawPicker.jsx

```diff
@@ -41,9 +56,13 @@ const PublicLawPicker = ({
         <div className="public-law-picker__container">
             <Picker
                 className="public-law-picker"
-                icon={<FontAwesomeIcon icon="scroll" size="sm" />}
+                icon={<FontAwesomeIcon icon="scroll" size="sm"/>}
```

Comment:
(Same lint error as above)
```suggestion
                icon={<FontAwesomeIcon icon="scroll" size="sm" />}
```